quickstarts/gradle/micronaut-customized-image/build.gradle

```diff
@@ -0,0 +1,66 @@
+plugins {
+    id("com.github.johnrengelman.shadow") version "7.0.0"
```

Comment:
did this come from the micronaut bootstrap page/project?

Reply:
Yeah, let me try removing this

Reply:
No, I was just asking, probably Micronaut uses this to package the jar artifact. If you remove, make sure that the resulting application is runnable.

Reply:
You're right. seems like this plugin generates `-all.jar`. 
```
micronaut-customized-image : $ ls build/libs/
micronaut-customized-image-1.5.0-SNAPSHOT-all.jar
micronaut-customized-image-1.5.0-SNAPSHOT.jar
micronaut-customized-image-1.5.0-SNAPSHOT-runner.jar
```
If I remove this plugin from `build.gradle`, I only see last 2 jars none of which are runnable